### PR TITLE
Update the adoption tables

### PR DESCRIPTION
**Pull request by ktoso**
cc @slashmo @adam-fowler

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ Compatible `Tracer` implementations:
 
 | Library | Status                     | Description |
 | ------- |----------------------------| ----------- |
-| [@slashmo](https://github.com/slashmo) / [**OpenTelemetry** Swift](https://github.com/slashmo/opentelemetry-swift) | Update for 1.0 in progress | Exports spans to OpenTelemetry Collector; **X-Ray** & **Jaeger** propagation available via extensions. |
-| [@pokrywka](https://github.com/pokryfka) / [AWS **xRay** SDK Swift](https://github.com/pokryfka/aws-xray-sdk-swift) | Not updated for 1.0        | ... |
+| [@slashmo](https://github.com/slashmo) / [**OpenTelemetry** Swift](https://github.com/slashmo/opentelemetry-swift) | 🟡 Update for 1.0 in progress | Exports spans to **OpenTelemetry Collector**; **Zipkin**, **X-Ray** and **Jaeger** compatible. |
```
Comment (slashmo) [inline README.md:30]:
```suggestion
| [@slashmo](https://github.com/slashmo) / [Swift **OTel**](https://github.com/slashmo/swift-otel) | 🟡 Update for 1.0 in progress | Exports spans to [**OpenTelemetry Collector**](https://opentelemetry.io/docs/collector/); Compatible with **Zipkin**, **X-Ray** **Jaeger**, and more. |
```

Comment (slashmo) [reply README.md:30]: I wouldn't limit the OTel Collector to those three systems.

File: README.md [hunk 2]
```diff
@@ -37,12 +37,19 @@ If you know of any other library please send in a [pull request](https://github.
 
 As this API package was just released, no projects have yet fully adopted it, the following table for not serves as reference to prior work in adopting tracing work. As projects move to adopt tracing completely, the table will be used to track adoption phases of the various libraries.
 
-| Library                  | Integrates     | Status                                                |
+| HTTP Servers/Frameworks  | Integrates     | Status                                                |
 |--------------------------|----------------|-------------------------------------------------------|
-| AsyncHTTPClient          | Tracing        | Pending                                               |
-| Swift gRPC               | Tracing        | Pending                                               |
+| [Hummingbird](https://github.com/hummingbird-project/hummingbird) | Tracing | 🟢 Built-in support |
```
Comment (slashmo) [inline README.md:42]: 🥳

**Review by slashmo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ktoso) [reply README.md]: Good call

**Pull request merged**